### DISCUSSION TITLE
Feedback for Translation of Korean Discussions

### DISCUSSION THREAD

**Discussion by bigzoo** [community/community#4372] [Discussions]
Hello! :wave:
This is a discussion for gathering user feedback on the beta release of Korean translations for Discussions.

To learn more about this feature and how to use it, see the [changelog post](https://github.blog/changelog/2021-07-19-machine-translation-for-discussions-content/).

Please use this discussion to share your experience using the new feature, and any ideas you have on how the experience could be improved.

**Comment by bryantson** [comment]
안녕하세요. 여기서 자유롭게 새로운 깃허브 디스커션 한글 번역에 관한 의견을 나눠주시길 바랍니다 ^^ 🇰🇷

**Reply by jihoonpython** [reply]
개선한다! 매일 새롭게 만들고 사용자의 의견을 듣는다.

**Reply by jihoonpython** [reply]
죄송합니다 잘못썼어요

**Reply by Student-Jasons** [reply]
오?! 깃허브가 한국어 번역을 하네요?? 근데 써보니 정확하진않네요 구글 번역api기준으로 바꾸면 좋겠습니다

**Reply by KangSuzy** [reply]
번역체 자체도 사실 엉성한 감에 좋아하는데도 
기대 이상으로 번역 수준이 높은 것 같아 좋아요 :D

**Reply by gyounghwa83** [reply]
오오 깃허브 한국어 번역 나이스!!
방금 가입했는데 기대됩니다.^~^

**Reply by kimdaehee0824** [reply]
아직은 부자연스러운 거 같습니다. 위에 분이 말하신 것처럼 구글 번역이나 파파고 번역 API 사용하면 더 좋을 거 같습니다.

**Reply by happy-spark** [reply]
번역체가 완벽하지는 않지만, 너무 좋은 기능이네요ㅎㅎ

**Reply by xeounxzxu** [reply]
너무 좋은 기능입니다! 잘 사용하겠습니다~!!

**Reply by jn821k** [reply]
대체로 좋습니다.😄

**Reply by jn821k** [reply]
very goooooooood

**Reply by jasuil** [reply]
👍 very useful!
I wrote some bug report in English.
After translating this with Korean, It works well.

**Reply by jkjun4574** [reply]
@ceohockey60

**Reply by ruukr8080** [reply]
피드백은 아니지만 감사를 전하고싶어서 댓글 남깁니다..
덕분에 너무 유용하게 잘 쓰고있어요~ 감사합니다~

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by aeongdesu** [comment]
상당히 좋은 기능인것 같습니다! 🎉 혹시 다른 주제이긴 하지만 이슈에도 지원을 하실 계획이 있으신가요?

**Reply by bryantson** [reply]
@KeepSOBP 님. 피드백 감사합니다. Product 팀에게 물어보고 미래에 계획에 있는지 알려 드리겠습니다.

**Reply by zeke** [reply]
Hi @KeepSOBP 👋🏼 

Thank you for your question. We are rolling out this feature as a beta in a limited number of languages for now (Korean and Brazilian Portuguese) to gather feedback and learn about whether this is useful for GitHub users. Depending on feedback and usage, we may expand to either more languages within Discussions or to other parts of GitHub's platform, e.g. Issues. To help us make that decision, please try this feature out, tell your friends who speak Korean, and give us your feedback! :slightly_smiling_face:

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by minpeter** [comment]
meetup에서 보고 사용해 보았는데 번역 퀄리티도 좋은것같습니다.
추가로는 docker, Adonis와 같은 명사는 번역을 하지 않고 그대로 표현하는것도 좋아보입니다 :)

**Reply by bryantson** [reply]
@minpeter 님, 피드백 감사합니다. 소중한 의견 깃허브 팀에게 알려드리겠습니다.

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by SeokminHong** [comment]
아주 편리한 기능 같습니다! 다만, 링크 인식에 대해 버그가 있는 것 같습니다. 링크 전후에 다른 텍스트들이 있거나 한국어 조사가 삽입되는 경우, 해당 단어들이 링크로 간주되는 경우가 있는 것 같습니다.

https://github.com/vercel/next.js/discussions/27352 링크의 번역을 보면 첫 문단에서 조사 "에"가 링크로 간주되고, 맨 아래 단락의 unordered list에는 "수학적으로"라는 단어가 링크로 인식되었습니다.

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by jihoonpython** [comment]
개량한다! 매일매일 사용자의 의견을 제시합니다.

**Reply by bryantson** [reply]
감사합니다

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by jihoonpython** [comment]
죄송합니다잘못썼어요를 기분이 좋아요로 인식되어있다

**Reply by bryantson** [reply]
감사합니다

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by lee-jongwoo** [comment]
<img width="616" alt="스크린샷 2021-10-23 오후 9 04 57" src="https://user-images.githubusercontent.com/70620386/138555302-7545d9b3-5aed-4bce-b9f2-c5449ffe21cf.png">

--------

이모티콘을 사용하면 글자 크기가 바뀝니다. 개발자 도구로 확인해 보니 이모티콘이 들어갈 자리에 텍스트가 들어가네요

-------

Using emojis cause the font size to change. DevTools shows me text inside the emoji area, which appears to be causing this issue.

**Reply by bryantson** [reply]
감사합니다

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by dayo05** [comment]
상당히 좋은 기능인데 이건 고쳐주셨으면 좋겠습니다.

tensor.ToString()에서의 .을 코드가 아닌 마침표로 인식합니다. 그로 인하여 번역된 결과가 좀.... 그렇습니다.

**Reply by bryantson** [reply]
네 감사합니다!

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by timo-kang** [comment]
좋네요!

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by KlartNET** [comment]
사랑해요 깃허브

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by ceohockey60** [comment]
Thank you all for your feedback. Looks like the machine translation quality is quite good.

What do you think about adding this feature to other parts of GitHub, like Issues, PR's, etc.?

**Reply by sappho192** [reply]
I'm really looking forward to use this feature in Issues and PRs.
Even though I can understand and write the English, sometimes there are some sentences that I have hard time understanding it.

Also, it would be nice if you add support for Japanese language (sorry if it's already supported).

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by gbeag** [comment]
![image](https://user-images.githubusercontent.com/96607123/151644534-8173767f-4f02-4de4-9d1d-61a72e9cba46.png)
번역 퀄리티를 더 높혀주세요!

**Reply by KlartNET** [reply]
한국어로 치면 "섭버ㅇㅐ서 온라인 유저 어떻게 구해요" 이런 오타 투성이라 번역 퀄리티의 문제는 아닌 것 같습니다...

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by revi** [comment]
번역 추천 기능을 끌 수 있는 스위치는 제공되지 않나요? 필요한 사람도 있겠지만 저는 굳이 번역이 필요하지 않아서 거슬리기만 합니다.

**Reply by revi** [reply]
번역 설정 여부 제공을 고려할 의사가 있는지 궁금합니다...

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by ultrakeypoint** [comment]
와 기능은 정말 대단하고 유용한 기능입니다.

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by norhu1130** [comment]
매우 유용합니다

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by keyfrog-21K** [comment]
된다면 깃허브 UI 자체를 모두 한국어로 바꾸는것도 고려해볼 만 한 것 같습니다.

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by happy-spark** [comment]
정말 좋은 기능이라고 생각합니다.👍

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by kuckjwi0928** [comment]
매우 유용한 기능입니다! 앞으로가 기대가되네요!!

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by jaehafe** [comment]
awesome

**Comment by jn821k** [comment]
좋은편입니다.

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by sappho192** [comment]
I'm really surprised that the Korean translation is very accurate. Maybe it's more natural than Google translator in some point.
This feature can make maintainers to use Discussions more!

**Comment by rsrrrrrr** [comment]
프로그램완성하고 푸쉬했다 보고 따라 들오옴 유용한것을 만들어 주어 감사합나다. 번역 이 자연스럽습니다.

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by ghost** [comment]
번역 퀄리티는 지속적인 개선이 필요하겠지만 해당 기능만으로 두세 번의 클릭을 하나로 줄일 수 있어서 좋다고 생각합니다.✨

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by oct-sky-out** [comment]
일반 텍스트로 작성된 외국어의 번역 퀄리티는 읽기 불편함 없이 좋으나, 마크다운으로 작성한 구문이 가끔 깨져서 해당 구문에 대해서 읽기 불편할 때가 간혹 발생했습니다.

아래는 마크다운 문법이 깨진 예입니다.

![스크린샷 2023-11-15 오후 11 40 47](https://github.com/community/community/assets/60546275/74ccb1cb-2c12-48e8-978b-b728635135b7)

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by museekee** [comment]
번역은 좋지만
![image](https://github.com/user-attachments/assets/b6427481-1307-452c-835c-403fcfbf0182)
이런 원본이
![image](https://github.com/user-attachments/assets/1a36c73a-8fe4-486f-871b-8e2377f276ed)
이런식으로 번역이 되는 점이 안타깝습니다

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by eclipse0922** [comment]
단순히 글 내용 기준으로만 번역하는 것이 아니라 해당 리포지토리의 컨텍스트 정보를 사용하면 좋을 것 같습니다.

일부 라이브러리나 언어, 리포지터리들은 이미 존재하는 단어들과 동일한 알파벳을 가지고 다른 의미로 사용합니다.
예 rust, carbon
서만 보면 이러한 이름들이 일반 단어들과 차이가 없게 느껴지지만
리포지터리의 컨텍스트 정보를 알고 있다면 그 단어들이 라이브러리를 지칭하는 것임을 바로 알 수 있을 것으로 보입니다.
해당되는 단어들은 번역을 하지 않고 그대로 표시하면 좋을 것 같습니다.

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by sinokadev** [comment]
번역 버튼을 눌러도 원문이 표시되는 버그가 있는 것 같습니다.
저는 지금 Kubuntu 24.04를 사용 중이고, firefox를 flatpak에서 다운로드 받아 사용중입니다.
혹시 저와 같은 버그를 겪고 있는 분이 있을까요?

**Reply by norhu1130** [reply]
GitHub에서 이 기능을 Deprecated 한 것으로 보입니다.
macOS 26.1, Safari 환경에서 다음과 같이 재현할 수 있었습니다:

https://github.com/localization/translations - 410,
{"error": "This translation service has been deprecated and is no longer available. Please use alternative translation methods."}

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by seuimi** [comment]
번역 기능자체는 모든 UI가 영어인 깃헙에서 토론이나 이런 부분들을 따로 웹사이트 번역없이 볼 수 있다는 점은 좋지만
지금은 작동을 안하는거 같네요.....

**Reply by bryantson** [reply]
안녕하세요. 한때 미국 깃허브에서 Senior Solutions Engineer 로 일하면서 이 기능을 확장하기 위해 여러모로 도움을 줄려고 했는데 위에 분이 말씀하신것 처럼 깃허브의 포커스가 코파일럿으로 바꾸면서 변화가 있었습니다. 깃 + 깃허브 관련 배우고 싶으시다면 코스를 제가 유데미로 출시한게 있으니 추천 드립니다:
https://www.udemy.com/course/git-course-korea/?referralCode=A010A6BCE799DAE0B589

그리고 한국인들이 연결 될수 있는 IT 그룹에 관심 있으시면 제가 만들고 이끄는 KAITA.ORG 로 오시길 추천 드립니다. 카카오 단톡방도 있으니 초청 가능합니다.

**Comment by norhu1130** [comment]
<img width="1341" height="552" alt="image" src="https://github.com/user-attachments/assets/04b987e1-c863-4e9a-917e-0e5a6f6ef1b9" />
@bryantson 모든 코멘트에 답글을 추가하실 경우 이메일이 스팸되는 문제가 있습니다,
가능하실 경우 별도의 코멘트를 추가 부탁드립니다 ;-;
